### PR TITLE
add link title everywhere

### PR DESCRIPTION
**Pull request by venturewise**
Note that I could not add a title to the PDF viewer's <iframe>, as it comes from the external integration.
## Issue

Fix #247
## Test URLs
  
- Before: https://main--merative2--hlxsites.hlx.page/company
- After: https://247-lighthouse--merative2--hlxsites.hlx.page/company
  
## Description
fix linked icons so they have a title.

### CODE REVIEW

File: scripts/scripts.js [hunk 1]
```diff
@@ -166,6 +166,22 @@ function buildDocumentUrl(main) {
...
+  spans.forEach((span) => {
+    const a = span.closest('a');
+    if (a) {
+      // all icons have a classlist of 'icon icon-name'
+      const iconName = span.classList[1];
+      if (iconName) {
+        a.title = iconName;
+      } else {
+        a.title = 'icon';
+      }
```
Comment (arumsey) [inline scripts/scripts.js:180]:
```suggestion
      a.title = iconName ? iconName : 'icon';
```

Comment (venturewise) [reply scripts/scripts.js:180]: Suggestions welcome. It was enough to squelch the Lighthouse error.

File: scripts/scripts.js [hunk 2]
```diff
@@ -166,6 +166,17 @@ function buildDocumentUrl(main) {
   }
 }
 
+function addTitletoIconLinks(main) {
```
Comment (arumsey) [inline scripts/scripts.js:169]: add captial `T`. `addTitleToIconLinks`

File: scripts/scripts.js [hunk 3]
```diff
@@ -166,6 +166,17 @@ function buildDocumentUrl(main) {
...
+  main.querySelectorAll('span.icon').forEach((span) => {
+    const a = span.closest('a');
+    if (a) {
+      // all icons have a classlist of 'icon icon-name'
+      const iconName = span.classList[1];
+      a.title = iconName || `${a.href}`;
```
Comment (arumsey) [inline scripts/scripts.js:175]: String literal not needed here. `a.href` will achieve the same thing.

**Pull request closed without merging**